.eslintrc

```diff
@@ -1,6 +1,7 @@
 {
   "extends": [
     "google",
+    "eslint:recommended",
```

Comment:
I don't know if this was intentional but there were a lot of rules that weren't enabled, like `no-undef` I added this just to catch a bunch of bugs for myself. Can remove it if that's what ya'll want